src/routes/game/components/elements/playerHandCard/PlayerHandCard.tsx

```diff
@@ -85,80 +73,6 @@ export const PlayerHandCard = (props: HandCard) => {
     }, LONG_PRESS_TIMER);
   };
 
-  const handleOnMouseDown = () => {
-    startPressTimer();
-    return;
-  };
-
-  const handleOnMouseUp = () => {
-    clearTimeout(timerRef.current);
-    return;
-  };
-
-  const handleOnTouchStart = () => {
-    startPressTimer();
-    handleMouseEnter();
-    return;
-  };
-
-  const handleOnTouchEnd = () => {
-    clearTimeout(timerRef.current);
-    handleMouseLeave();
-    return;
-  };
-
-  const handleMouseEnter = () => {
-    if (ref.current === null || !canPopUp) {
-      return;
-    }
-    const rect = ref.current.getBoundingClientRect();
-    const xCoord = rect.left < window.innerWidth / 2 ? rect.right : rect.left;
-    const yCoord = rect.top < window.innerHeight / 2 ? rect.bottom : rect.top;
-    dispatch(
-      setPopUp({
-        cardNumber: card.cardNumber,
-        xCoord: xCoord,
-        yCoord: yCoord
-      })
-    );
-  };
-
-  const handleMouseLeave = () => {
-    dispatch(clearPopUp());
-  };
-
-  const degree = handSize > 1 ? lerp(-15, 15, cardIndex / (handSize - 1)) : 0;
```

Comment:
Removed the fanning. Not sure if this fanning of cards is a great idea anyway particularly for mobile devices?

Reply:
I liked the card fanning but I did have some people say they weren't a fan (boom boom)

I'm happy for it to be removed, we can add it back in later